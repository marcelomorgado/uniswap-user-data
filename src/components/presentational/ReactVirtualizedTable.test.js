import React from "react";
import { shallow } from "enzyme";
import ReactVirtualizedTable from "./ReactVirtualizedTable";

it("renders without crashing", () => {
  expect(shallow(<ReactVirtualizedTable />)).toMatchSnapshot();
});
