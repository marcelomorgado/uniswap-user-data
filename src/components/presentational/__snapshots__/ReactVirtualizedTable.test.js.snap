// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<WithStyles(Paper)
  style={
    Object {
      "height": 400,
      "width": "100%",
    }
  }
>
  <WithStyles(MuiVirtualizedTable)
    columns={
      Array [
        Object {
          "dataKey": "dessert",
          "flexGrow": 1,
          "label": "Dessert",
          "width": 200,
        },
        Object {
          "dataKey": "calories",
          "label": "Calories (g)",
          "numeric": true,
          "width": 120,
        },
        Object {
          "dataKey": "fat",
          "label": "Fat (g)",
          "numeric": true,
          "width": 120,
        },
        Object {
          "dataKey": "carbs",
          "label": "Carbs (g)",
          "numeric": true,
          "width": 120,
        },
        Object {
          "dataKey": "protein",
          "label": "Protein (g)",
          "numeric": true,
          "width": 120,
        },
      ]
    }
    onRowClick={[Function]}
    rowCount={200}
    rowGetter={[Function]}
  />
</WithStyles(Paper)>
`;
